surefire-its/src/test/resources/pojo-simple/src/test/java/PojoTest.java

```diff
@@ -21,6 +21,11 @@
 {
     private static boolean setUpCalled = false;
```

Comment:
`false` is default value. Not needed here. 
The field is written once anyway.

Reply:
I understand it's reference to [4.12.5 Initial Values of Variables](https://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5), but is that comment informational only, or shall I remove explicit false?